### ISSUE TITLE
Emoji are rendered inside Markdown code blocks

### ISSUE THREAD

**Issue by zoriya** [jpanther/congo#876]
### Issue description

Using code blog inside markdown content files do not escape emoji conversion.

For example: `ffmpeg -map 0:a:0` converts `:a:` to :a:

This is because hugo's `emojify` transform does not know about markdown code blogs, it was fixed on hugo's side by disabling their internal emojify transform and using the markdown's renderer one (see https://github.com/gohugoio/hugo/issues/4978 for more info).

Please note that disabling hugo's emojify is not possible (to my knowledge) since congo hardcodes `| emojify` calls everywhere. This means the config option `enableEmoji = false` does nothing.

### Theme version

v2.8.2

### Hugo version

hugo v0.124.1+extended linux/amd64 BuildDate=unknown VendorInfo=nixpkgs

### Which browser rendering engines are you seeing the problem on?

Chromium (Google Chrome, Microsoft Edge, Brave, Vivaldi, Opera, etc.), Firefox (Mozilla Firefox), WebKit (Safari)

**Comment by jpanther** [comment]
I don't believe this is an issue with Congo as this is the expected behaviour of Hugo's markdown parser. Have you tried this Hugo configuration?
```toml
enableEmoji = false

[markup.goldmark.extensions]
emoji = true
```

**Comment by zoriya** [comment]
Yes I did. The issue comes from the `| emojify` inside Congo's source code. Those pipes don't read the configuration and use the builtin emoji parser that doesn't understand emoji.

The fix is to remove the emojify pipe everywhere markdown content is expected and let glowfish handle emoji for us.
